src/act.drive.cpp

```diff
@@ -2031,6 +2037,7 @@ ACMD(do_switch)
   ch->vfront = !ch->vfront;
   snprintf(buf, sizeof(buf), "$n climbs into the %s.", ch->vfront ? "front" : "back");
   act(buf, TRUE, ch, 0, 0, TO_ROOM);
+  look_at_room(ch, 0, 0)
```

Comment:
A check here would be good too! IIRC it's something along the lines of `if (!PRF_FLAGGED(ch, PRF_SCREENREADER))`